# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/clang/hotfix-no-clang-doc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- clang-21.1.2.src/clang-tools-extra/CMakeLists.txt.vanilla	2025-09-16 17:19:13.610225268 +0200
+++ clang-21.1.2.src/clang-tools-extra/CMakeLists.txt	2025-09-16 17:19:19.005883182 +0200
@@ -21,7 +21,7 @@
 add_subdirectory(clang-tidy)
 
 add_subdirectory(clang-change-namespace)
-add_subdirectory(clang-doc)
+#add_subdirectory(clang-doc)
 add_subdirectory(clang-include-fixer)
 add_subdirectory(clang-move)
 add_subdirectory(clang-query)
